src/command/dev/mod.rs

```diff
@@ -91,7 +90,7 @@ pub struct SupergraphOpts {
         long = "supergraph-config", 
         conflicts_with_all = ["subgraph_name", "subgraph_url", "subgraph_schema_path"]
     )]
-    supergraph_config_path: Option<Utf8PathBuf>,
+    supergraph_config_path: Option<FileDescriptorType>,
```

Comment:
It seems that here we modelled the config path as a Utf8PathBuf, but for `compose` we modelled it as a `FileDescriptorType`. I've gone with FileDescriptorType here so that does give the potential to allow `supergraph.yaml`'s to be pipe in from stdout should people want to do that.

Reply:
huh! nice!